### DISCUSSION TITLE
Can zitadel be configured with Grafana?

### DISCUSSION THREAD

**Discussion by aurabhi** [zitadel/zitadel#5100] [General]
If yes, then is it possible to map Zitadel orgs with grafana orgs? If there are any reference please share.

**Comment by fforootd** [comment]
Well the two things i am aware of are https://github.com/zitadel/zitadel/issues/3037#issuecomment-1397031160 and https://discord.com/channels/927474939156643850/927474939156643853/1012706024878248067
